packages/@aws-cdk/aws-codepipeline-actions/lib/codebuild/build-action.ts

```diff
@@ -8,6 +8,7 @@ import { Action } from '../action';
 // keep this import separate from other imports to reduce chance for merge conflicts with v2-main
 // eslint-disable-next-line no-duplicate-imports, import/order
 import { Construct } from '@aws-cdk/core';
+import { CodeCommitSourceAction } from '../codecommit/source-action';
```

Comment:
This should be moved up, just below the `import { BitBucketSourceAction } from '..';` line.